### ISSUE TITLE
License

### ISSUE THREAD

**Issue by hwestbrook** [orempel/twiboot#5]
I'd like to use this code in a product I am developing, but I cannot accept the GPL 2 license attached. 

Are you open to changing the license? LGPL?

**Comment by orempel** [comment]
No, twiboot will stay GPL-2 as it is already used in other projects as well.
